diff-containers/src/Data/Map/Diff/Strict.hs

```diff
@@ -1,286 +1,33 @@
-{-# LANGUAGE DeriveAnyClass             #-}
-{-# LANGUAGE DeriveFoldable             #-}
-{-# LANGUAGE DeriveFunctor              #-}
-{-# LANGUAGE DeriveGeneric              #-}
-{-# LANGUAGE DerivingStrategies         #-}
-{-# LANGUAGE FlexibleInstances          #-}
-{-# LANGUAGE GeneralisedNewtypeDeriving #-}
-{-# LANGUAGE LambdaCase                 #-}
-{-# LANGUAGE Rank2Types                 #-}
-{-# LANGUAGE ScopedTypeVariables        #-}
-{-# LANGUAGE ViewPatterns               #-}
-
 module Data.Map.Diff.Strict (
-    Diff (..)
+    -- * Types
+    Diff
   , DiffEntry (..)
-  , DiffHistory (..)
-  , NEDiffHistory (..)
-    -- * Conversions between empty and non-empty diff histories
-  , nonEmptyDiffHistory
-  , toDiffHistory
-  , unsafeFromDiffHistory
+    -- * Conversion
+  , keysSet
     -- * Construction
   , diff
+    -- ** Maps
+  , fromMap
+  , fromMapDeletes
+  , fromMapInserts
+    -- ** Lists
   , fromList
   , fromListDeletes
-  , fromListEntries
   , fromListInserts
-  , singleton
-  , singletonDelete
-  , singletonInsert
-    -- * Predicates
+    -- * Query
+    -- ** Size
   , null
```

Comment:
Would it make sense to pick another name to avoid conflicts with `null`?

Reply:
I'm following the `Data.Map` naming conventions, and in general `Data.Map.Diff.Strict` should be qualified imported -- would you be okay with leaving it as is?

Reply:
That makes sense. Thanks.